Examples/OutlineViewExample/OutlineViewExample.xcodeproj/project.pbxproj

```diff
@@ -303,7 +303,7 @@
 					"@executable_path/../Frameworks",
 				);
 				MACOSX_DEPLOYMENT_TARGET = 11.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.OutlineViewExample;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.example.OutlineViewExample${SAMPLE-CODE-DISAMBIGUATOR}";
```

Comment:
This might have happened because of the pesky Xcode UI. You need underscores instead of dashes here.

```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = "com.example.OutlineViewExample${SAMPLE_CODE_DISAMBIGUATOR}";
```

The same applies to the second instance in this file. OutlineViewDraggingExample has the correct value.

Reply:
There is a second instance at Line:329.